### PR TITLE
Update set of "supported by" on main page

### LINKED ISSUE #6795: Doc updates
Quick scanning, the following should be corrected:

- "Does Dask have a SQL interface": yes it now does (if perhaps not fully featured)
- paid support: add Coiled? others (saturn)?
- main page Supported By: there was discussion on how to add logos here, but I would say Coiled, Saturn, CapitalOne, Prefect should be here. Also, an older page Funding should be the same with more details, or just removed.

### PR DESCRIPTION
**Pull request by martindurant**
There being more logos now, we could consider splitting into rows. I have gone for alphabetical ordering

Closes https://github.com/dask/dask/issues/6795

Note that prefect the software depends on/uses Dask, and prefect the company supports Dask - to contrast with NVIDIA/rapids, where the two have different names and logos.

### CODE REVIEW

**Comment by martindurant** [comment]
Screen Shot 2020-11-11 at 10 01 38

**Comment by martindurant** [comment]
Two-row version:
Screen Shot 2020-11-11 at 10 08 53

**Comment by mrocklin** [comment]
I think that two-row makes more sense

File: index.html [hunk 1]
```diff
@@ -525,6 +525,12 @@ <h2>Supported By</h2>
                 <div class="col supporter">
                   <a href="https://anaconda.com"><img src="https://upload.wikimedia.org/wikipedia/en/c/cd/Anaconda_Logo.png"></a>
                 </div>
+                <div class="col supporter">
+                  <a href="https://www.capitalone.com/"><img src="http://www.capitalone.co.uk/images/c1/brand/logo.svg"></a>
+                </div>
+                <div class="col supporter">
+                  <a href="https://coiled.io/"><img src="_images/coiled-logo.svg"></a>
+                </div>
```
Comment (mrocklin) [inline index.html:533]: Coiled logos are available here: https://github.com/coiled/logos

I recommend https://github.com/coiled/logos/blob/master/horizontal.svg

**Comment by mrocklin** [comment]
Thank you for setting this up @martindurant !

**Comment by quasiben** [comment]
I also prefer the two row version for what it's worth

**Comment by jrbourbeau** [comment]
+1 the two row version looks great

**Comment by jrbourbeau** [comment]
Thanks @martindurant!

**Pull request merged**